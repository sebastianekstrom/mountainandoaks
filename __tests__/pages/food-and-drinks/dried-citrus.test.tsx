import { render, screen } from "@testing-library/react";
import React from "react";

import DriedCitrus from "pages/food-and-drinks/dried-citrus";

describe("DriedCitrus page", () => {
	it("renders title correctly", () => {
		render(<DriedCitrus />);
		const title = screen.queryAllByText(/Dried citrus/i);
		expect(title).toBeDefined();
	});

	it("renders the ingress text", () => {
		render(<DriedCitrus />);
		const ingress = screen.queryAllByText(
			/Take your cocktails to the next level/i,
		);
		expect(ingress).toBeDefined();
	});

	it("renders the correct ingredients", () => {
		render(<DriedCitrus />);
		const ingredientList = screen.getByTestId("INGREDIENT_LIST");
		expect(ingredientList).toMatchSnapshot();
	});

	it("renders the method steps", () => {
		render(<DriedCitrus />);
		const step = screen.queryAllByText(
			/Slice the citrus fruit into thin rounds/i,
		);
		expect(step).toBeDefined();
	});
});
