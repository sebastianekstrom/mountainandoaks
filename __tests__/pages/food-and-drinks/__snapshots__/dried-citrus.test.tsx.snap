// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DriedCitrus page renders the correct ingredients 1`] = `
<section
  class="mb-8 lg:mb-0"
  data-testid="INGREDIENT_LIST"
>
  <div>
     
  </div>
  <div
    class="flex flex-col"
  >
    <ul>
      <li
        class="flex justify-between mb-1 transition-opacity "
      >
        <div
          class=""
        >
          <p
            class="variable font-sans text-paragraph text-regular leading-paragraph font-semibold"
          >
            Citruses of your choice
          </p>
        </div>
        <p
          class="variable font-sans text-paragraph text-regular leading-paragraph "
        >
          4
        </p>
      </li>
    </ul>
  </div>
</section>
`;
